app/controllers/events_controller.rb

```diff
@@ -1,8 +1,42 @@
 class EventsController < ApplicationController
 
+  def admin_show
+    authenticate_admin!
+    @event = Event.find_by_id(params[:id])
+  end
```

Comment:
hasn't this been merged with `show`? If not make a note somewhere to merge the two & display the content that needs displaying on the current show page.